content/en/apps/guides/data/csv-to-docs.md

```diff
@@ -186,6 +186,49 @@ The resulting doc would be as follows, with the `_id` from `district_1` as the `
   "_id": "45293356-353c-5eb1-9a41-baa3427b4f69"
 }
 ```
+##### Link to a parent document already existing in couchdb
+You may wish to link the new record to a parent document that already exists in couchdb that was created in the past. Get the UUID of the existing parent document and place it in the column named `parent._id` as shown below:
+
+| reference_id:excluded | parent._id                          | is_name_generated | name | reported_date:timestamp |
+| --------------------- | ------------------------------------| ----------------- | ---- | ----------------------- |
+| health_center_1       | 0c31056a-3a80-54dd-b136-46145d451a66| false             | HC1  | 1544031155715           |
+
+The resulting doc would be as follows:
+```json
+{
+  "type": "health_center",
+  "parent": {
+    "_id": "f223f240-5d6a-5a7a-91d4-46d3c59de73e"
+  },
+  "is_name_generated": "false",
+  "name": "HC1",
+  "reported_date": 1544031155715,
+  "_id": "45293356-353c-5eb1-9a41-baa3427b4f69"
+}
+```
+
+If the parent document has more parent documents that need to be tracked, then more columns created with names corresponding to the parent UUIDs as shown below:
+
```

Comment:
No it is not mandatory but it can be useful for ease of access especially while writing queries

Reply:
I believe an incomplete hierarchy may introduce some issues in the app

Reply:
Noted... Let me update the documentation to reflect that.

Reply:
Just checked. It worked with the custom hierarchy but did not work with the default hierarchy. Strange! Please try with a custom hierarchy config if possible to make sure that is the case.

Reply:
> Just checked. It worked with the custom hierarchy but did not work with the default hierarchy. Strange! Please try with a custom hierarchy config if possible to make sure that is the case.

@dianabarsan requesting for your insights into this.

Reply:
A couple of questions: 
1. Are you talking about json docs that you haven't uploaded yet or docs that you have uploaded?
2. Can you confirm this is happening with a small number of docs without uploading, just checking the resulting json-docs? 

I'm asking specifically because csv-to-docs doesn't require you to connect to an instance, it doesn't do any requests. So there's no way of it determining the lineage of an existing document.

Reply:
I think including the full hierarchy information in your CSV is going to be required in the end. can you please include an example of csv and config when your lineage got populated after upload?

Reply:
The json doc inside `json-docs` folder only has one parent information:
```json
  "parent": {
    "_id": "5221f46c-482c-402f-964b-b4fe326ac3ff"
  }
```

When we upload it to the instance using `upload-docs`, the resulting doc in the database has subsequent parent information auto-populated. However, this did not work in an instance having the default hierarchy.

```
  "parent": {
    "_id": "5221f46c-482c-402f-964b-b4fe326ac3ff",
    "parent": {
      "_id": "70cae1b5-7bc4-40fa-b34c-bed652703f0c",
      "parent": {
        "_id": "71b88e5d-4759-496e-99e0-376624a5581c",
        "parent": {
          "_id": "43e3765c-ff80-4e67-897f-4874bba6edd1",
          "parent": {
            "_id": "7335343c-7b54-4125-9f1d-b37d445f026a",
            "parent": {
              "_id": "3a289ad4-03d3-4bc4-909f-abf76db4e95a"
            }
          }
        }
      }
    }
  }
```

Reply:
Can you please share which transitions you have enabled on the instance?

Reply:
I think that what you're seeing is the `generate_patient_id_on_people` populating your lineage. Can you try enabling that transition on your default config and upload a new doc?

Reply:
It worked in the default hierarchy too after swapping the transitions.

Reply:
Ok, thanks for confirming. 
Hope it's clear now :)

Reply:
`generate_patient_id_on_people`? Why does it affect when uploading places only?

Reply:
it affects places and people. but the lineage being fixed is a side-effect of you having this transition enabled. Which means that you **have** to enter the whole lineage in the CSV.